share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx

```diff
@@ -68,28 +87,28 @@
           <gateTime>67</gateTime>
           </Articulation>
         <Articulation name="sforzato">
-            <velocity>150</velocity>
-            <gateTime>100</gateTime>
-        </Articulation>
+          <velocity>150</velocity>
+          <gateTime>100</gateTime>
+          </Articulation>
         <Articulation name="sforzatoStaccato">
-              <velocity>150</velocity>
-              <gateTime>50</gateTime>
-        </Articulation>
+          <velocity>150</velocity>
+          <gateTime>50</gateTime>
+          </Articulation>
         <Articulation name="marcatoStaccato">
-              <velocity>120</velocity>
-              <gateTime>50</gateTime>
-        </Articulation>
+          <velocity>120</velocity>
+          <gateTime>50</gateTime>
+          </Articulation>
         <Articulation name="marcatoTenuto">
-              <velocity>120</velocity>
-              <gateTime>100</gateTime>
-        </Articulation>
+          <velocity>120</velocity>
+          <gateTime>100</gateTime>
+          </Articulation>
         <Channel name="Soprano">
+          <controller ctrl="32" value="17"/>
```

Comment:
What is good for? Just curious...

Reply:
This is the result of saving the templates.

Reply:
Hmm, is that then a good reason to keep it?

Reply:
Is it a reason to remove it? When creating a soprano will save this line. So, why not having it in the template as well?

Reply:
I'd rather first understand what it does

Reply:
I think it switches to another bank in the Soundfont. Hoping it exists (it won't in the default one as far as I can tell?)

Reply:
then again we do something like this for 12 string guitar in instruments.xml, so this might be OK

Reply:
It actually is the bank for the expressive sounds

Reply:
Something that happens automatically anyway, so not needed here, but most probably not harmful either

Reply:
As a matter of fact, when creating a score using this template in 3.5.2 (or 3.2.3) and save the score, you see these same line popping up in the score.

Reply:
Yes, as said, happens automagically as ot the SND implementation